integration-tests/tests/src/index.ts

```diff
@@ -56,6 +56,7 @@ describe("Test Harness", () => {
 // Simplify once https://github.com/kraenhansen/mocha-remote/issues/58 gets solved
 describe.skipIf(environment.integration === false || environment.integration === "false", "Integration tests", () => {
   require("./tests");
+  require("./tests/credentials/email-password.ts");
```

Comment:
This should simply go into the `tests/index.ts` (which moved to `tests.ts` if you rebase your PR.